packages/sdks/snippets/angular/src/app/editable-regions/editable-regions.component.ts

```diff
@@ -0,0 +1,46 @@
+import { CommonModule } from '@angular/common';
+import { Component, type OnInit } from '@angular/core';
+import {
+  Content,
+  fetchOneEntry,
+  type BuilderContent,
+} from '@builder.io/sdk-angular';
+import {
+  CustomColumnsComponent,
+  customColumnsInfo,
+} from './custom-columns/custom-columns.component';
+
+@Component({
+  selector: 'app-editable-region',
+  standalone: true,
+  imports: [CommonModule, Content, CustomColumnsComponent],
+  template: `
+    <div *ngIf="content">
+      <builder-content
+        [content]="content"
+        [model]="model"
+        [apiKey]="apiKey"
+        [customComponents]="customComponents"
+      />
+    </div>
+    <div *ngIf="notFound">404</div>
+  `,
+})
+export default class EditableRegionComponent implements OnInit {
```

Comment:
we can remove `implements OnInit`

Reply:
Ah! Though it works fine, after reading the Angular docs, one issue in [github](https://github.com/angular/angular/issues/5814) and other posts in Stackoverflow, it is more of the best practices. TS will help us with typos during compile time. 

If you also see the below images, if I implement **_onInit_**, when I mistype **_ngOnInit_**(), it throws an error. If we don't implement it, Angular compiles fine and gives no error, and then we will have difficulty figuring it out. 

<img width="909" alt="Screenshot 2024-10-26 at 9 55 48 AM" src="https://github.com/user-attachments/assets/309a3696-fe86-40f1-a458-4500bfaa055b">
<img width="751" alt="Screenshot 2024-10-26 at 9 56 02 AM" src="https://github.com/user-attachments/assets/3c873e1d-a3f7-41dd-b855-4d952b46e932">

That is the reason why it is there in the code. Let me know if it makes sense :)

Reply:
sure, do you think we should also add this to other components to maintain consistency?